packages/instantsearch.js/src/lib/__tests__/InstantSearch-test.tsx

```diff
@@ -474,43 +476,99 @@ See https://www.algolia.com/doc/api-reference/widgets/configure/js/`);
   });
 
   describe('insights middleware', () => {
-    test('does not add insights middleware by default', () => {
-      const search = new InstantSearch({
-        searchClient: createSearchClient(),
-        indexName: 'test',
+    const createSearchClientWithInsights = () =>
```

Comment:
It's long, but it's 100% unambiguous.

```suggestion
    const createSearchClientWithAutomaticInsightsOptedIn = () =>
```